tests/views/test_api.py

```diff
@@ -1,29 +1,326 @@
-from mock import patch
```

Comment:
I know this is kind of a pain, but could you add a short docstring to the beginning of each test? in pytest if it fails it will fail with the docstring message, which is easier to read than a traceback with the function name in underscore_case